sample_formatted_code.do_not_merge.c

```diff
@@ -0,0 +1,562 @@
+// Formatted using
```

Comment:
Yes, I know these are C++ style comments. This is not meant to be committed, imho. I can always fix the commend style up if some form of this should be committed.